.github/workflows/rust.yml

```diff
@@ -19,9 +19,9 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - name: Build
-      run: cargo build
+      run: cargo build --all-features
```

Comment:
Ah, we will also need to test with no features, otherwise only the `noline` codepath is built/tested.

Reply:
Perhaps do what the clippy tasks does: all-features, no-default-features and *the default*